src/main/scala/mesosphere/marathon/event/http/HttpEventActor.scala

```diff
@@ -2,53 +2,91 @@ package mesosphere.marathon.event.http
 
 import akka.actor._
 import akka.pattern.ask
+import mesosphere.marathon.api.v2.json.Formats._
 import mesosphere.marathon.event._
+import mesosphere.marathon.event.http.HttpEventActor._
 import mesosphere.marathon.event.http.SubscribersKeeperActor.GetSubscribers
-import play.api.libs.json.{ JsValue, Json }
+import mesosphere.marathon.metrics.Metrics
 import spray.client.pipelining.{ sendReceive, _ }
 import spray.http.{ HttpRequest, HttpResponse }
 import spray.httpx.PlayJsonSupport
-import mesosphere.marathon.api.v2.json.Formats._
 
 import scala.concurrent.Future
-import scala.util.{ Failure, Success }
+import scala.concurrent.duration._
+import scala.util.Success
+import System.currentTimeMillis
+
+object HttpEventActor {
+  case class NotificationFailed(url: String)
+  case class NotificationSuccess(url: String)
+  case class NotificationSlow(url: String)
+
+  case class EventNotificationLimit(failed: Long, deadLine: Option[Deadline]) {
+    def nextFailed: EventNotificationLimit = {
+      val next = failed + 1
+      EventNotificationLimit(next, Some(math.pow(2, next.toDouble).seconds.fromNow))
+    }
+    def notLimited: Boolean = deadLine.fold(true)(_.isOverdue())
+    def limited: Boolean = !notLimited
+  }
+  val NoLimit = EventNotificationLimit(0, None)
+}
 
-class HttpEventActor(val subscribersKeeper: ActorRef) extends Actor with ActorLogging with PlayJsonSupport {
+class HttpEventActor(conf: HttpEventConfiguration, val subscribersKeeper: ActorRef, metrics: Metrics)
+    extends Actor with ActorLogging with PlayJsonSupport {
 
   implicit val ec = HttpEventModule.executionContext
   implicit val timeout = HttpEventModule.timeout
+  val pipeline: HttpRequest => Future[HttpResponse] = addHeader("Accept", "application/json") ~> sendReceive
+  var limiter = Map.empty[String, EventNotificationLimit].withDefaultValue(NoLimit)
 
-  val pipeline: HttpRequest => Future[HttpResponse] = (
-    addHeader("Accept", "application/json")
-    ~> sendReceive)
+  //metrics
+  val metricOutstandingRequests = metrics.counter("service.mesosphere.marathon.event.http.outstanding-requests")
```

Comment:
Usually, we use "metrics.name" to create the metrics name in combination with `MetricPrefixes`. I am really not sure if I like that approach but for consistency it would be better to stick to that, I guess.